package.json

```diff
@@ -30,7 +30,7 @@
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "graphql": "^14.0.0"
+    "graphql": "^15.0.0"
```

Comment:
let's make this "^14.5.0 || ^15.0.0"

Reply:
Updated the graphql version as per your suggestion to "^14.5.0 || ^15.0.0"

Reply:
you actually updated the `graphql` devDependency. I was suggesting you change the peerDependency to the above string. The dev dependency should remain unchanged